### PR TITLE
Add a section on how to remove backslash in interface names for eos-native paths

### PR DESCRIPTION
**Pull request by mpergament**
Add a section on how to remove backslash in interface names for eos-native paths

### CODE REVIEW

**Review by burnyd — APPROVED** [review]
LGTM!

**Review by burnyd — APPROVED** [review]

File: docs/telemetry/adapters/ocprometheus/index.md [hunk 1]
```diff
@@ -634,6 +634,43 @@ The minimum required flags for TerminAttr are:
 
 After configuring our switches we can head over to Grafana and start drawing our graphs.
 
+### Fix for multilane interfaces (EOS-native paths)
+
+If interface statistics streamed using eos-native paths, then backslash would be part of the
+interface names:
+![Multilane interface names](resources/multilane-before.png)
+
+Use following metric_relabel_configs configuration to remove backslash in interface names:
```
Comment (sulrich) [inline docs/telemetry/adapters/ocprometheus/index.md:643]: - s/Use/Use the/
- s/to remove/to remove the/

**Review by sulrich — COMMENTED** [review]
minor suggestion below. otherwise LGTM, thanks!

**Review by sulrich — APPROVED** [review]

**Review by sulrich — CHANGES_REQUESTED** [review]
can we remove the `.DS_Store` files from this?

**Review by sulrich — APPROVED** [review]
👍

**Pull request merged**